bats/fb-test-puppet.bats

```diff
@@ -54,15 +54,15 @@ fi
     modpath=/etc/puppet/environments/production/modules
   fi
 
-  if [ ! -d $modpath/ntp ]; then
-    puppet module install -i $modpath -v 4.2.0 puppetlabs/ntp
+  if [ ! -d $modpath/chrony ]; then
+    puppet module install -i $modpath -v 0.3.2 aboe/chrony
```

Comment:
Note I could not find NTP on EL8 as it is on EL7 so I switched this to chrony.

Reply:
Does it make sense to use something simple like motd? OTOH, the module looks simple enough and supports what we need. puppetlabs-motd looks more complex.